TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8298,7 +8298,8 @@ table_reference: single_table_or_view_reference;
 single_table_or_view_reference: tableOrViewOrCorrelationName correlation_clause?;
 correlation_clause: SQL_AS? (correlation_name=UserDefinedWord) new_column_names?;
 new_column_names: LeftParenthesisSeparator new_column_name (SQL_CommaSeparator new_column_name)* RightParenthesisSeparator;
-new_column_name: UserDefinedWord;
+new_column_name: column_name;
+column_name:UserDefinedWord;
```

Comment:
There is no need for a new definition. You can keep the previous definitions or rename `new_column_name` rule to `column_name` if you want to reuse 'column_name' in future rules.